tests/virt/cluster/service_account/test_service_account.py

```diff
@@ -46,10 +49,12 @@ def test_vm_with_specified_service_account(service_account_vm):
     assert pod_sa == vm_namespace, "ServiceAccount should be attached to the POD"
 
     # Verifies that ServiceAccount is attached to VMI
+    # Change mount device based on cluster cpu architecture
+    mount_device = "/dev/vdb" if get_nodes_cpu_architecture(nodes=list(Node.get())) == S390X else "/dev/sda"
```

Comment:
please create a session scope fixture `is_s390x_cluster` and reuse it instead of calling all the time `get_nodes_cpu_architecture`